.github/workflows/dev_environment.yml

```diff
@@ -24,14 +24,9 @@ on:
         required: false
         type: boolean
         default: false
-      registry_cache_update:
+      update_registry_cache:
```

Comment:
A description on this might parameter be nice to help decipher what types of strings are valid inputs here (especially since this same parameter in deployments.yml is a boolean).

Reply:
Yes, I meant to document all parameters once the pipelines are stable - which they are now. If you don't mind, I'll make a separate PR, since this one is needed to restore the build, and it would be good to have this build overnight so that tomorrow the build is restored.

Reply:
Sounds good to me.